feat(list-lookup) : add list lookup query logic

cain4chain opened the pull request:
## Description of Changes

feat(list_lookup): add query file for list lookup provider method

## Linked Issues / Tickets

https://github.com/4chain-ag/go-overlay-services/issues/14

Describe the tests you've added or any testing steps you've taken.

feat(list_lookup_test): add list lookup provider test file for success, empty, nil

- [x] I have added new unit tests
- [x] All tests pass locally
- [x] I have tested manually in my local environment

## Checklist:

- [x] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have run the linter

cain4chain committed:
feat(list-lookup) : add list lookup query logic

feat(list_lookup): add lookup query file
feat(list_lookup_test): add lookup test file for success, empty, nil

github-actions[bot] committed:
chore: update generated files

cain4chain committed:
chore(lookup_list) : Fix line spacing issue

chore(lookup_list) :  Fix line spacing issue

github-actions[bot] committed:
chore: update generated files

cain4chain committed:
Merge branch 'main' into FEATURE-Add-lookup-provider-query-handler-support

mgosek-4chain reviewed (commented):
Please apply the same hints as described in PR #58

cain4chain committed:
Merge branch 'main' into FEATURE-Add-lookup-provider-query-handler-support

cain4chain committed:
feat(lookup_list_test) : update based on review

chore(jsonutils) : Add json utils method for decode
chore(misc) : Update test code formatting
